apps/aecore/src/aec_conductor.erl

```diff
@@ -1122,6 +1122,10 @@ handle_successfully_added_block(Block, Hash, Events, State, Origin) ->
         {micro_changed, State2 = #state{ consensus = Cons }} ->
             {ok, setup_loop(State2, false, Cons#consensus.leader, Origin)};
         {changed, BlockType, NewTopBlock, State2} ->
+            Origin == block_created andalso
```

Comment:
I see this changes the report to be done only by the miner that produced the block. Why? Do we intend to run this on our non-mining seed nodes?

Reply:
Hmm, this shouldn't be the case. It should post info for each block that's "appended" to the node database. Including received from sync/p2p gossip.